app/models/metric/helper.rb

```diff
@@ -154,16 +176,16 @@ def self.get_time_zone(options)
     options[:tz] || TimeProfile::DEFAULT_TZ
   end
 
-  def self.get_time_range_from_offset(start_offset, end_offset = nil, options = {})
+  def self.time_range_from_offset(start_offset, end_offset, tz)
     # Get yesterday at 23:00 in specified TZ. Then convert that to UTC. Then subtract offsets
-    tz = Metric::Helper.get_time_zone(options)
+    tz ||= Metric::Helper.get_time_zone(nil)
```

Comment:
why pass nil to `get_time_zone`?  why not simply `tz ||= Metric::Helper.get_time_zone`?